gdshelpers/helpers/small.py

```diff
@@ -47,6 +48,20 @@ def id_to_alphanumeric(column, row):
     return int_to_alphabet(row) + str(int(column))
 
 
+def parse_alphanumeric(text):
+    """
+    Do the reverse of `id_to_alphanumeric`.
+    Returns the (column, row) tuple.
+    """
+    regex = "([A-Z]*)[^0-9]*([0-9]*)"
```

Comment:
what do we need `...[^0-9]...` for? to match the description of the function this part is not necessary
```suggestion
    regex = "([A-Z]*)([0-9]*)"
```

Reply:
I don't remember exactly where I encountered the "A-1" format instead of "A1", but if it should only be used as the inverse of id_to_alphanumeric then we don't need this, right